next.config.js

```diff
@@ -1,6 +1,6 @@
 /** @type {import('next').NextConfig} */
 
-const basePath = process.env.NODE_ENV === 'production' ? '/openbc-web' : '';
+const basePath = process.env.NODE_ENV === 'production' ? '' : '';
```

Comment:
I'm open to removing the `basePath` variable altogether if we want, but figured I would leave it in here in-case we ever want to use it again in the future.

Since we no longer have a site deployed on GitHub Pages, I figured this would be fine to modify.

Reply:
yup, let's remove it.

Reply:
Addressed in [31aa915](https://github.com/CodeForBc/codeforbc-web/pull/2/commits/31aa9150c461278922ceab10c5dd2763274157e6)